vizro-core/docs/pages/user_guides/components.md

```diff
@@ -37,5 +37,12 @@ listed below to fill your dashboard with visuals.
     Use containers to organize your page components into subsections inside the dashboard.
 
     [:octicons-arrow-right-24: View User Guide](container.md)
+- :material-tab-plus:{ .lg .middle } __Tabs__
```

Comment:
Something is a little strange here

![image](https://github.com/mckinsey/vizro/assets/83698606/6c204eb6-97a8-466b-a21c-0f092ae1b17a)

Reply:
Thanks! Fixed 👍